build/build-sample-reports.js

```diff
@@ -18,6 +18,12 @@ const lhr = /** @type {LH.Result} */ (require('../lighthouse-core/test/results/s
 const {LH_ROOT} = require('../root.js');
 const htmlReportAssets = require('../lighthouse-core/../report/report-assets.js');
 
+const flowResult = /** @type {LH.FlowResult} */ (JSON.parse(
```

Comment:
I believe since `parse` returns any, you can put the `type` comment on the line before (cleaner syntax than the parentheses for manually coercing it)